scripts/release/create_minor_release_PR_commit.sh

```diff
@@ -29,26 +30,34 @@ done
 LAST_VERSION=$(echo "$LAST_UPDATE_FILE" |cut -d '-' -f 1 |cut -d '/' -f 2)
 
 echo "CURRENT_VERSION is $CURRENT_VERSION"
-#echo "LAST_UPDATE_FILE is $LAST_UPDATE_FILE"
 echo "LAST_VERSION is $LAST_VERSION"
 echo "RELEASE_BRANCH is $RELEASE_BRANCH"
 echo "NEW_VERSION is $NEW_VERSION"
 cd ~/"$SOURCES_DIR"/"$FORK_DIR"
 
 
 # Derived Variables
-#RELEASE_PR_BRANCH="release-$NEW_VERSION-$RELEASE_BRANCH"
-RELEASE_PR_BRANCH="release-$NEW_VERSION"
+RELEASE_PR_BRANCH="release/release-$NEW_VERSION"
```

Comment:
Might be good to use something like "release/$NEW_VERSION-to-main", to distinguish it from the branch for the PR that goes into the release branch itself.

Reply:
Edited branch names accordingly.